openwisp_users/tests/test_admin.py

```diff
@@ -493,6 +493,44 @@ def test_organization_uuid_field(self):
         html = '<input type="text" name="name" value="default"'
         self.assertNotContains(response, html)
 
+    def test__action_active(self):
+        user = User.objects.create(is_active=False)
```

Comment:
please don't do it this way, creating a user like that (without any info set except `is_active`) generates an invalid user into the DB, writing tests on an invalid model instance is really fragile.

Use this method to create an user, https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/tests/utils.py#L31 then flag it as inactive.